server/controllers/enjoy.go

```diff
@@ -0,0 +1,47 @@
+package controllers
+
+import (
+	"net/http"
+
+	"server/graphql"
+	"server/utils"
+
+	"github.com/gin-gonic/gin"
+)
+
+type EnjoyParams struct {
+	City string `json:"city" binding:"required"`
+	Page int    `json:"page" binding:"required,gte=0"`
+}
```

Comment:
Greater or equal 1 for pagination
Use form instead of json. Swagger generates documentation with small case, So this will be a problem for front client.
```suggestion
	City string `form:"city" binding:"required"`
	Page int    `form:"page" binding:"required,gte=1"`
}
```

Reply:
I fix it in another branch